source/GameAction.cpp

```diff
@@ -189,6 +190,13 @@ void GameAction::LoadSingle(const DataNode &child)
 		if(maxDays < minDays)
 			swap(minDays, maxDays);
 		events[GameData::Events().Get(child.Token(1))] = make_pair(minDays, maxDays);
+	} else if (key == "play") {
+		if (hasValue) {
+			music = child.Token(1);
+			playMusic = true;
+		} else {
+			child.PrintTrace("Error: play doesn't have a specified file. If you want to stop music, do 'play \"\"'");
+		}
 	}
```

Comment:
We don't need specific error messages for each keyword that is missing a value. Just let it fall past this if statement if the value is missing and allow the generic error message to be sent.
```suggestion
	}
	else if(key == "play" && hasValue)
		music = child.Token(1);
```

Reply:
done